src/main/resources/alfresco/core-services-context.xml

```diff
@@ -1352,4 +1352,11 @@
       <property name="policyComponent" ref="policyComponent" />
       <property name="namespaceService" ref="namespaceService" />
    </bean>
+
+   <bean id="deploymentMethodProvider" class="org.alfresco.repo.deployment.DeploymentMethodProviderImpl" >
+      <property name="deploymentMethodAsString">

```

Comment:
I think that Spring will do the conversion.  Have you checked?

Reply:
Yes you are right, Spring does indeed do the conversion. I'll change it in the next commit.